### ISSUE TITLE
some request not working

### ISSUE THREAD

**Issue by alyfreym** [davidteather/TikTok-Api#751]
I use with custom_verifyFp.

1 - _api.get_user_
2 - _api.by_username_ 

`Bad request.
We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner`

where I found secUID ?

**Comment by pierrehabte2017** [comment]
Same Issue with api.get_user

```
TikTokApi.exceptions.TikTokCaptchaError: TikTok blocks this request displaying a Captcha
Tip: Consider using a proxy or a custom_verifyFp as method parameters
```

**Comment by fs-akjha** [comment]
Using custom_verifyFp for api.by_username(), removing cookies and generating new s_v_webid but still getting 
**Bad request. We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner
TikTokApi.exceptions.TikTokCaptchaError: TikTok blocks this request displaying a Captcha
Tip: Consider using a proxy or a custom_verifyFp as method parameters**
Please help me out here, any help will be really appreciated.

**Comment by MuffinFox** [comment]
The requests are blocked by cloudflare because we use python requests lib. Needs a rewrite to use headless browser.

**Comment by MuffinFox** [comment]
Duplicate of  #750

**Comment by Alex-Laparra** [comment]
until david updates it, use @BeatRoy 's pull request guys, fixed it for me

**Comment by davidteather** [comment]
Updated with V4.1.0
